src/components/ItemView.vue

```diff
@@ -14,6 +14,18 @@ const code = computed(() => `import xarray as xr
 
 xr.open_dataset("${ item?.assets?.data?.href }", engine="zarr")`)
 
+function generateHTMLReferences(references: string[]): string[] {
+  return references.map(reference => {
+    if (reference.match('^https?://')) {
+      return `<a href="${reference}">${reference}</a>`;
+    } else if (reference.match(/^10\.\d+\//)) {
+      return `<a href="https://doi.org/${reference}">${reference}</a>`;
+    }
+    return `<p>${reference}</p>`;
```

Comment:
Using arbitrary external strings as raw html is generally and [in particular](https://vuejs.org/guide/essentials/template-syntax.html#raw-html) unsafe, as this allows anyone to run arbitrary code in the security domain ("origin") of our site and must be avoided. It's ok to use `v-html` if the entire string is properly checked not to include malicious html, but that's not the case here.

Reply:
Thanks for the catch and link!